test/acceptance/todo-cli-test.js

```diff
@@ -340,6 +340,75 @@ describe('todo usage', () => {
       expect(todoDirs).toHaveLength(0);
     });
 
+    it('errors if a todo item has expired when running without params', async function () {
+      project.setConfig({
+        rules: {
+          'require-button-type': true,
+        },
+      });
+
+      project.write({
+        app: {
+          templates: {
+            'require-button-type.hbs': '<button>Check Expiration</button>',
+          },
+        },
+      });
+
+      project.writeTodoConfig({
+        error: 5,
+      });
+
+      // generate todo based on existing error
+      await run(['.', '--update-todo'], {
+        // change the date so errorDate is before today
+        env: {
+          TODO_CREATED_DATE: subDays(new Date(), 10).toJSON(),
+        },
+      });
+
+      // run normally and expect an error for not running --clean-todo
+      let result = await run(['.']);
+
+      expect(result.exitCode).toEqual(1);
+      expect(result.stdout).toMatchInlineSnapshot(`
+        "app/templates/require-button-type.hbs
+          1:0  error  All \`<button>\` elements should have a valid \`type\` attribute  require-button-type
+          -:-  error  Expired todos exist. Please run \`ember-template-lint --clean-todo\` to remove expired todos.  expired-todo-rule
+
+        ✖ 2 problems (2 errors, 0 warnings)
+          2 errors and 0 warnings potentially fixable with the \`--fix\` option."
+      `);
+
+      // run clean-todo, and expect that this will delete the expired todo item
+      await run(['app/templates/require-button-type.hbs', '--clean-todo']);
+
+      // run normally again and expect only the error
+      result = await run(['.']);
+
+      let todoDirs = fs.readdirSync(getTodoStorageDirPath(project.baseDir));
+
+      expect(result.exitCode).toEqual(1);
+      expect(result.stdout).toMatchInlineSnapshot(`
+      "app/templates/require-button-type.hbs
+        1:0  error  All \`<button>\` elements should have a valid \`type\` attribute  require-button-type
+
+      ✖ 1 problems (1 errors, 0 warnings)
+        1 errors and 0 warnings potentially fixable with the \`--fix\` option."
+    `);
```

Comment:
This snapshot will need to be updated once you remove the additional error you added for expired.